src/qgsquick/qgsquickutils.h

```diff
@@ -103,6 +103,11 @@ class QgsQuickUtils: public QObject
         const QgsCoordinateTransformContext &context,
         const QgsPointXY &srcPoint );
 
+    /**
+      * Calculates the conversion factor between the specified distance units.
+      */
+    Q_INVOKABLE static double distanceFromUnitToUnitFactor( const QgsUnitTypes::DistanceUnit fromUnit, const QgsUnitTypes::DistanceUnit toUnit );
```

Comment:
Let's add this to a new class `UnitTypes` (or similar) to keep the API a bit tidier.